config/main.go

```diff
@@ -166,7 +166,7 @@ func NewTestConfig() Config {
 	c.electrs.port = "3002"
 
 	c.rpcServer.host = os.Getenv("ADDR")
-	c.rpcServer.port = "18443"
+	c.rpcServer.port = "18433"
```

Comment:
Why it's changing?

Reply:
Nigiri's liquid node default port: https://github.com/vulpemventures/nigiri/blob/e02c2fdd0d93a8014df1f5dcf1027f97e88689fc/cli/constants/constants.go#L33